src/components/editable-input/editable-input.jsx

```diff
@@ -25,6 +25,10 @@ class EditableInput extends React.Component {
 
   onEdit = () => {
     this.setState({ originalValue: this.state.value, editing: true });
+
+    if (this.props.onEdit) {
+      this.props.onEdit();
```

Comment:
you might want to pass the onEdit event to the function? like we do with onChange, onSubmit. Not sure if its required, but at least its consistent.

Reply:
That is very true! Added it now :)